_includes/imaging-tutorial-etcher-banner.html

```diff
@@ -1,5 +1,6 @@
 {% include icon.html type="info" %}
 There's an easier way to image your SD card! Check out
 [the "Getting Started" guide](/docs/getting-started#step-2-copy-the-image-on-to-the-sd-card){: .alert-link }
-for a cross-platform guide.
+for a cross-platform guide. This tutorial isn't being maintained and may not
```

Comment:
We no longer supply `.zip`s, so the Win32DiskImager tutorial is broken (others may also have issues with that -- I can't remember). I played around with having a separate "danger" banner on that one page, but it looked distracting and awkward. Overall my thinking was essentially that we hadn't been giving these pages much attention so I would rather not confuse people; if you would like to continue supporting them, I can rejigger this but we will need to take the Windows one into consideration.

Statistics-wise, it looks like many people find the Ubuntu guide from outside sources; I suspect that Google has picked it up and people are following it for images other than ev3dev as well. Past that, I don't have much data to go off of right now (or the know-how to persuade Google to give it to me) so I can't make a data-driven call on numbers of people choosing these guides over the Etcher option.